app/javascript/gobierto_data/lib/router.js

```diff
@@ -17,20 +17,24 @@ export const router = new VueRouter({
       component: Main,
       children: [{
         path: "",
-        component: Index,
-        props: { activeSidebarTab: 0 }
+        component: Index
       }, {
         path: ":id/:tab?",
         component: Dataset,
+        name: 'Dataset',
         // send props as a function, default to tab[0], otherwise take the index
-        props: ({ params: { tab = tabs[0] } }) => ({ activeDatasetTab: tabs.indexOf(tab), activeSidebarTab: 1 })
+        props: ({ params: { tab = tabs[0] } }) => ({ activeDatasetTab: tabs.indexOf(tab) })
       }, {
         path: ":id/q/:queryId?",
         component: Dataset,
-        // active the editor for queries
-        props: { activeDatasetTab: 1, activeSidebarTab: 1 }
+        name: 'Dataset',
+        props: { activeDatasetTab: 1 }
       }]
-    }]
+    }],
+    scrollBehavior() {
```

Comment:
Why is this here?

Reply:
Because I read it in the [documentation](https://router.vuejs.org/guide/advanced/scroll-behavior.html#async-scrolling), where should I be?

Reply:
Aaaaaaamigo, it's a predefined method. Ok, then. Cool.